index.html

```diff
@@ -47,9 +47,9 @@
         for title in response.css('h2.entry-title'):
             yield {'title': title.css('a ::text').extract_first()}
 
-        next_page = response.css('div.prev-post > a ::attr(href)').extract_first()
+        next_page = response.css('div.prev-post > a')
         if next_page:
-            yield scrapy.Request(response.urljoin(next_page), callback=self.parse)
+            yield response.follow(next_page[0], self.parse)
```

Comment:
I must admit the example on how to get an attribute using CSS is compelling too.